src/wallet/wallettool.cpp

```diff
@@ -103,6 +105,207 @@ static void WalletShowInfo(CWallet* wallet_instance)
     tfm::format(std::cout, "Address Book: %zu\n", wallet_instance->m_address_book.size());
 }
 
+/* End of headers, beginning of key/value data */
+static const char *HEADER_END = "HEADER=END";
+/* End of key/value data */
+static const char *DATA_END = "DATA=END";
+
+static bool SalvageWallet(fs::path path)
```

Comment:
nit: `const fs::path&=

Reply:
This commit is part of #18918 so comments should be left there.